skatepark/block-templates/search.html

```diff
@@ -0,0 +1,27 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"layout":{"inherit":true}, "tagName":"main"} -->
+<main class="wp-block-group">
+
+	<!-- wp:search {"label":""} /-->
+
+	<!-- wp:spacer {"height":45} -->
+		<div style="height:45px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
+
+	<!-- wp:query {"queryId":1,"query":{"perPage":10,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true}} -->
+	<!-- wp:query-loop -->
```

Comment:
I think if you change this to `post-template` instead of `query-loop` it will fix the warning, e.g. https://github.com/Automattic/themes/blob/trunk/blockbase/block-templates/search.html#L8-L13

Reply:
Thanks @jffng! I've just pushed this, hopefully it fixes the warning 🤞